docs/In-Band Manageability User Guide - UCC.md

```diff
@@ -138,20 +138,36 @@ UCC CA file (*.pem.crt):
 /home/abc/ucc.ca.pem.crt
 ```
 
-10. If the cloud provisioning is successful, the following message
+10. A Yes/No proxy prompt appears asking if a proxy will be used.  Select y if a proxy needs to be configured.
+```
+Configure a proxy?  [Y/N] y
+```
+
+11. Configure the proxy information if selected above.
+
+```commandline
+Please enter the hostname:
+proxy-dmz.intel.com
+
+
+Please enter the server port (default 1883):
+911
+
+```
+12. If the cloud provisioning is successful, the following message
     appears:
 ```
 Successfully configured cloud service!
 ```
 
-11. A Yes/No user prompt appears asking for a certificate verification
+13. A Yes/No user prompt appears asking for a certificate verification
```

Comment:
For now UCC does not plan on supporting OTA updates.
Lets auto select 'N' without prompting the user for a selection.

Reply:
@pillaiabhishek - I was able to get it to skip the question altogether for UCC.  updated the documentation to show that too.